app/presenters/account_show_presenter.rb

```diff
@@ -18,7 +18,9 @@ def show_password_reset_partial?
   end
 
   def show_pii_partial?
-    decrypted_pii.present? || user.identity_verified?
+    decrypted_pii.present? || user.identity_verified? unless
+      user.pending_in_person_enrollment.present? || user.
+        gpo_verification_pending_profile.present?
```

Comment:
The combination of `||` operators plus the `unless` keyword is pretty disorienting. I'd suggest either implementing this as an early return or consistently using only boolean operators `&&` `||`.

Also, should we use something like `user.pending_profile?` instead, to avoid having to manage all of the different pending states. For example, would we also want to hide this on fraud review pending? Password reset pending?

```suggestion
    return false if user.pending_profile?
    decrypted_pii.present? || user.identity_verified?      
```

Reply:
This logic was revised in ac9737e087.